src/hooks.js

```diff
@@ -152,7 +152,7 @@ export function useClickOutside(cb, deps) {
     return () => {
       document.removeEventListener('click', handleClick, true)
     }
-  }, [...deps, handleClick])
+  }, [deps, handleClick])
```

Comment:
I don't think we need to pass any dependencies to `useClickOutside`, we should just depend on the `cb` changing to re-run any effects here.